nilearn/regions/region_extractor.py

```diff
@@ -169,6 +170,13 @@ def connected_regions(maps_img, min_region_size=1350,
         index_of_each_map.extend([index] * len(regions))
         all_regions_imgs.extend(regions)
 
+    if len(all_regions_imgs) == 0:
+        warnings.warn("No regions are found to be extracted. Check if the "
+                      "given 'maps_img' is not empty. "
+                      "Returning image as it is.", stacklevel=2)
+        input_img = new_img_like(maps_img, maps_img.get_data())
```

Comment:
>However, this may break what happens next, for example if you try to plot it with prob_atlas. We have to think about it.

I thought the same and felt that returning given image as it is would be safe and valid.